src/liballoc/arc.rs

```diff
@@ -971,6 +972,44 @@ unsafe impl<#[may_dangle] T: ?Sized> Drop for Arc<T> {
     }
 }
 
+impl Arc<Any + Send + Sync> {
+    #[inline]
```

Comment:
Attributes tend to go below doc comments

Reply:
This is a copy of the corresponding code in `Rc`.